dapp-test/massa-wallet-provider.js

```diff
@@ -0,0 +1,266 @@
+(function(f){if(typeof exports==="object"&&typeof module!=="undefined"){module.exports=f()}else if(typeof define==="function"&&define.amd){define([],f)}else{var g;if(typeof window!=="undefined"){g=window}else if(typeof global!=="undefined"){g=global}else if(typeof self!=="undefined"){g=self}else{g=this}g.wallet = f()}})(function(){var define,module,exports;return (function(){function r(e,n,t){function o(i,f){if(!n[i]){if(!e[i]){var c="function"==typeof require&&require;if(!f&&c)return c(i,!0);if(u)return u(i,!0);var a=new Error("Cannot find module '"+i+"'");throw a.code="MODULE_NOT_FOUND",a}var p=n[i]={exports:{}};e[i][0].call(p.exports,function(r){var n=e[i][1][r];return o(n||r)},p,p.exports,r,e,n,t)}return n[i].exports}for(var u="function"==typeof require&&require,i=0;i<t.length;i++)o(t[i]);return o}return r})()({1:[function(require,module,exports){
```

Comment:
Aren't you supposed to use the last generated one ?

Reply:
I have now removed this file. It is not necessary to have a js file in the dapp-test folder. The dapp-test project is an npm project that has this command:

  "scripts": {
    "serve": "cp ../bundle.js ./bundle.js && http-server --cors --port 9009"
  },
  
  so basically, running server would copy the bundle.js file from the wallet-provider lib and server the html file which is using it. I have added a .gitignore file which ignores the bundle.js.

Reply:
The way to run is:  `npm run build` at the root of the library which will build the bundle.js and then `cd dapp-test && npm run serve` as explained above. Output is currently in the console.

Reply:
Does the readme contain this information ?